### PR TITLE
CLC-4014 Remove Research Hub support

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-4014

This should just do what it says, but I wouldn't mind a quick walkthrough with @johncrossman or @raydavis, if available, on our workflow for release instructions and config changes.

### CODE REVIEW

File: src/assets/stylesheets/_icons.scss [hunk 1]
```diff
@@ -48,9 +48,6 @@
 .cc-icon-oskicat {
   background-image: url('../images/icon_oskicat_16x16.png');
 }
-.cc-icon-researchhub {
-  background-image: url('../images/icon_research_16x16.png');
```
Comment (christianv) [inline src/assets/stylesheets/_icons.scss]: @paulkerschen can we also remove the actual icon (if it's not used anywhere else)

Comment (paulkerschen) [inline src/assets/stylesheets/_icons.scss]: @christianv Oh, yes, thanks for catching that.

**Comment by raydavis** [comment]
Thanks for thinking of that, @paulkerschen . I generally edit the configurations for Bamboo and for Dev, and then edit the Release-Specific Instructions to match.

Of course, my attempt from this morning didn't work so well! :)

**Comment by raydavis** [comment]
Oh, and in the specific case of _removing_ properties, we sometimes delay removal for a release so as to enable rollbacks, builds of unmerged branches, etc. That may be most appropriate in this case.

**Pull request merged**